rules/opentitan/defs.bzl

```diff
@@ -124,10 +124,8 @@ CLEAR_TEST_CMD = " "
 def _parameter_name(env, pname):
     if not pname:
         (_, suffix) = env.split(":")
-        if "cw310" in suffix:
-            pname = "cw310"
-        elif "cw340" in suffix:
-            pname = "cw340"
+        if "fpga" in suffix:
+            pname = "fpga"
```

Comment:
I wonder if we should try use cw310/cw340 if they are present, and then try "fpga" as a fallback. If we decide in the future to future merge fpga_params and silicon_params, then it can try cw310->fpga->common.

Reply:
So `fpga` would only be a fallback and not merged with `cw310`/`cw340`?
I think that could work, but I'll do it in a follow up.

Reply:
I've added an extra commit which makes `cw310` the fallback parameter name if `fpga` isn't provided, but I've only written it to avoid merge skew errors and it's not intended as a full solution.

We can revisit this later if we want to fallback behaviour properly.